app/components/autocomplete/autocomplete.js

```diff
@@ -76,16 +110,17 @@ export default class Autocomplete extends PureComponent {
                 containerStyle.push(style.borders);
             }
         }
-
         return (
             <View style={wrapperStyle}>
                 <View style={containerStyle}>
                     <AtMention
+                        listHeight={this.listHeight()}
```

Comment:
really minor... what if you do something like
```
const calculatedHeight = this.listHeight();
```

and then pass `calculatedHeight` as a prop to both components? that way you run the function just once instead of twice

Reply:
@enahum Minor but still you're absolutely correct. Changing momentarily.